app/public/js/services/firebaseService.js

```diff
@@ -1,5 +1,9 @@
 angular.module('FirebaseService', []).service('firebaseService', function($rootScope) {
 
+  this.setUserPassword = function(newPassword) {
+    return $rootScope.user.updatePassword(newPassword);
```

Comment:
Yeah, but all the logic is abstracted in the user itself, which is something you know only when you know firebase. When I look at this file, I shouldn't ask myself if the user uses firebase, it should either be written in a comment, or very explicit in the code.